blocks/bricks/__init__.py

```diff
@@ -335,24 +335,30 @@ class LinearMaxout(Initializable):
     -----
     See :class:`Initializable` for initialization parameters.
 
-    .. todo:: Name of :attr:`linear_transformation` shouldn't be hardcoded.
-
     """
     @lazy
     def __init__(self, input_dim, output_dim, num_pieces, **kwargs):
         super(LinearMaxout, self).__init__(**kwargs)
+        self.linear_transformation = Linear()
```

Comment:
It is 100% up to, but I find `transformation` super long to type. `linear_brick` or just `linear`?